wavefront/server/modules/agents_module/agents_module/services/async_agentic_execution_result_consumer.py

```
@@ -0,0 +1,153 @@
+                    _POLL_COUNT,
+                    _BLOCK_MS,
+                )
+
+                for _stream_name, entries in messages:
+                    for msg_id, fields in entries:
+                        try:
+                            await self._process(fields)
+                            await asyncio.to_thread(
+                                self._cache.xack, _STREAM, _GROUP, msg_id
+                            )
+                        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)